internal/saucecloud/worker.go

```diff
@@ -0,0 +1,10 @@
+package saucecloud
+
+import "github.com/saucelabs/saucectl/internal/job"
+
+type result struct {
```

Comment:
Put this into into a temporary spot that is `worker`, so that it's shared between the runners. May likely get refactored further with upcoming PRs.